distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/SampleNsiliClient.java

```diff
@@ -442,6 +478,7 @@ public void testStandingQueryMgr(POA poa, Query query) throws Exception {
             start.at(ResultDAGConverter.getAbsTime(startDate));
 
             LifeEvent end = new LifeEvent();
+            final long ONE_YEAR = 365L * 24L * 60L * 60L * 1000L;
```

Comment:
You don't need to change it, but for future reference, you can do: TimeUnit.DAYS.toMillis(365).